src/open_inwoner/components/templates/components/Step/StepIndicator.html

```diff
@@ -1,7 +1,7 @@
-{% load link_tags %}
+{% load link_tags i18n %}
 
 {% spaceless %}
-    <aside class="step-indicator">
+    <aside class="step-indicator" aria-label="{% trans "Stappen indicator" %}">
```

Comment:
I think a more useful label would be "step 1 out of 3" then hide the rest of the contents using aria-hidden?